.coveragerc

```diff
@@ -4,6 +4,7 @@ source =
     convert2rhel
 omit =
     convert2rhel/unit_tests/*
+    convert2rhel/pkgmanager/handlers/base.py
```

Comment:
I don't care that our coverage goes down tbh. This shouldn't be excluded as we might do something in the future and forget that we excluded the file

Reply:
Looks reasonable to me as well... But if codecov complains about missing coverage, should we do a unit test for this class?

Reply:
You can mark a class as not needing tests AFAIK. Otherwise we can ignore for now